InfiniteRecharge/src/main/java/frc/robot/Buttons.java

```diff
@@ -0,0 +1,60 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
+import edu.wpi.first.wpilibj2.command.button.POVButton;
+import frc.robot.operatorinterface.PS4Constants;
+
+public class Buttons {
+
+    Joystick driverControl = new Joystick(0);
+    Joystick operatorControl = new Joystick(1);
+
+    // drive
+    JoystickButton driveAutoAim = new JoystickButton(driverControl, PS4Constants.CIRCLE.getId());
+
+    int driveSpeed = PS4Constants.LEFT_STICK_Y.getId();
+    int driveTurn = PS4Constants.RIGHT_STICK_X.getId();
+
```

Comment:
Do we want to name these driveSpeedAxis and driveTurnAxis to make it clear they aren't buttons?